clients/benchmarks/CMakeLists.txt

```diff
@@ -92,3 +92,53 @@ set_target_properties( hipblaslt-bench PROPERTIES
 add_dependencies( hipblaslt-bench hipblaslt-common )
 
 rocm_install(TARGETS hipblaslt-bench COMPONENT benchmarks)
+
+# Other bench files
+add_executable( hipblaslt-bench-groupedgemm-fixed-mk client_groupedgemm_fixed_mk.cpp)
```

Comment:
suggest following the file name

Reply:
I followed the hipblaslt-bench with client.cpp and hipblaslt-bench